j4l13n/pos_ncd_tracking
# Add SMS Auth Token field to General Settings

# Pull Request: Add SMS Auth Token Field to General Settings

## Overview

This pull request introduces the SMS Auth Token field to the General Settings of the Odoo module. The new field allows users to store and manage their SMS authentication tokens more effectively.

## Changes Made

- Inherited the `res.config.settings` view to add a new field for the SMS Auth Token.
- Positioned the SMS Auth Token field at the end of the General Settings form for improved accessibility and visibility.
- Wrapped the new field within a group labeled "SMS Settings" for better organization within the settings interface.
  
## Motivation

Adding the SMS Auth Token field enhances the functionality of the General Settings by allowing administrators to easily configure SMS authentication settings directly from the settings interface.

## How to Test

1. Update the Odoo module with the changes from this pull request.
2. Navigate to the General Settings.
3. Verify that the SMS Auth Token field appears at the end of the settings form.
4. Test saving a value in the SMS Auth Token field to ensure it saves correctly.

## Checklist

- [ ] Tested on the appropriate Odoo version.

## Additional Notes

Please review the changes and provide feedback or approval for merging this pull request. Thank you!
